scripts/mkcloud

```diff
@@ -993,7 +1016,7 @@ function rebootcrowbar()
 {
     # reboot the crowbar instance
     #  and test if everything is up and running afterwards
-    sshrun "reboot"
+    sshrun "reboot >/dev/null &"
```

Comment:
@tpatzig not for us :) It feels strange that this fixes an issue for you. I'd really like to reproduce it.
Did you verify that this is really the cause for a blocking you observed? At least the `>/dev/null` should not be needed.